packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animation.css

```diff
@@ -72,14 +76,14 @@
     }
     .animation-container {
         padding-top: 7vh;
-        height: 45vh;
+        height: 100vh;
```

Comment:
Same here and below. Please remove the height:100vh from all media queries except for the max-width:1300px (because this css will make no difference)

Reply:
Remove height from max-width as well